js/common.js

```diff
@@ -166,50 +166,56 @@ export function registerEventsToCard(element) {
 	element.addEventListener("dragend", e => handleDragEnd(e), false);
 }
 
+/**
+ * @param {Request | string} input
+ * @returns {Promise<Response>}
```

Comment:
戻り値は `Promise<any>` では（toot の json を返してるため）
```suggestion
 * @returns {Promise<any>}
```